airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -63,10 +64,22 @@
 
   private final ConfigRepository configRepository;
   private final SchedulerJobClient schedulerJobClient;
+  private final SpecFetch specFetch;
 
-  public SchedulerHandler(final ConfigRepository configRepository, SchedulerJobClient schedulerJobClient) {
+  public SchedulerHandler(ConfigRepository configRepository, SchedulerJobClient schedulerJobClient) {
+    this(
```

Comment:
NIT: can you use the same convention (new line for every argument) in the DestinationHandler ctor?